readme.txt

```diff
@@ -3,7 +3,7 @@ Contributors: equalizedigital, alh0319, stevejonesdev
 Tags: accessibility, accessible, wcag, ada, WP accessibility
 Requires at least: 6.2
 Tested up to: 6.6.7
```

Comment:
```suggestion
Tested up to: 6.6.2
```

Reply:
I updated this to 6.7 since we have tested it a lot with the RC. Upping to 6.7 will close this auto issue:  https://github.com/equalizedigital/accessibility-checker/issues/788